project1/application.py

```diff
@@ -35,25 +35,54 @@ def register():
         var=request.form.get("email")
         var1=request.form.get("psw")
         timestamp=datetime.datetime.now()
-        user = User(email=var,password=var1,timestamp=timestamp)
+        users1= User.query.all()
+        for user in users1:
+            if var==user.email:
+                return "<h2 Style='color: red;text-align:center'>You already have registered !Please Login </h2>"       
```

Comment:
You should add after checking for blank fields. Please move this code to line #49